### PR TITLE
Fix issue in email notification when override email is not set

### PR DESCRIPTION
**Pull request by praveenkuttappan**
- Add trace for each email notification
- Fix issue when override email address for testing is empty. Currently request is sent to logic app with empty email address.

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@
 using Microsoft.ApplicationInsights;
 using System.Net.Http;
 using System.Net.Http.Json;
+using Newtonsoft.Json;
```
Comment (benbp) [inline src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs:19]: Why newtonsoft over System.Text.Json? Or is newtonsoft already pretty heavily used in this project?

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs:19]: It's already used. I will change it to use System.Text.Json. Thanks

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs]: Done.

Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs]: System.Text.Json has some issues when used with cosmos db.

**Pull request merged**